DEVELOPER.md

```diff
@@ -419,3 +426,79 @@ docker tag wind-river/cloud-platform-deployment-manager:debug ${MY_REGISTRY}/win
 docker push ${MY_REGISTRY}/wind-river/cloud-platform-deployment-manager:latest
 docker push ${MY_REGISTRY}/wind-river/cloud-platform-deployment-manager:debug
 ```
+
+## Troubleshooting dep
+If you encounter issues with dep (eg, dep command hangs), check the following:
+
+#### ~/.cache directory
+
+Your home directory is a shared file system under /folk. Having the .cache
+directory in your folk directory can end up filling up the disk quota. Use a
+symbolic link instead.
+
+```bash
+mv /folk/${USER}/.cache /localdisk/loadbuild/${USER}/.cache and
+ln -s /localdisk/loadbuild/${USER}/.cache /folk/${USER}/.cache
+```
+
+#### Hanging processes
+
+Ctrl-C may not completely terminate dep. Check ps -a for dep and its
```

Comment:
I think you'd want to note here that, if the "dep ensure -update" command is hung, using "ps" to look for potentially hung child processes is a great step, before killing the hung dep command.

Reply:
Fixed